### DISCUSSION TITLE
Onboarding PR commits being overwritten

### DISCUSSION THREAD

**Discussion by viceice** [renovatebot/renovate#8039] [Request Help] [answered]
This bug still happens to me on GitLab.

Every time I push a commit on the onboarding merge request, and it re-runs, the branch is then rebased by it. I can't have custom configuration...

_Originally posted by @Songkeys in https://github.com/renovatebot/renovate/issues/1961#issuecomment-745765112_

**Comment by viceice** [comment, marked as answer]
@SongKeys Do you try to ammend the git commit? Renovate uses the commit author to check moduification, so you need to push a separat commit.

**Reply by songkeys** [reply]
Ah.. I know the issue here then. I used the same git author to push the commit. That's my fault. Sorry.
